tests/drivers/counter/counter_basic_api/src/test_counter.c

```diff
@@ -26,6 +26,13 @@ void *exp_user_data = (void *)199;
 struct counter_alarm_cfg alarm_cfg;
 struct counter_alarm_cfg alarm_cfg2;
 
+#define INST_DT_COMPAT_LABEL(n, compat) DT_LABEL(DT_INST(n, compat)),
+/* Generate a list of LABELs for all instances of the "compat" */
+#define LABELS_FOR_DT_COMPAT(compat) \
```

Comment:
Having an equivalent test dedicated macro (or set of test macros) that could be applied more generally would be nice.

Reply:
I figured as we see a pattern occur we can refactor this to a common place.